tests/system/action/meeting_user/test_set_data_delegation.py

```diff
@@ -159,6 +159,31 @@ def test_delegated_to_error_user_cannot_delegate_has_delegations_himself(
             response.json["message"],
         )
 
+    def test_reverse_delegation(
+        self,
+    ) -> None:
+        self.set_models(
+            {
+                "meeting_user/14": {"vote_delegations_from_ids": [13]},
+                "meeting_user/13": {
+                    "vote_delegated_to_id": 14,
+                    "vote_delegations_from_ids": [],
+                },
+            }
+        )
+        response = self.request_executor(
+            {"vote_delegations_from_ids": [], "vote_delegated_to_id": 13}
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
+            "meeting_user/14",
+            {"vote_delegated_to_id": 13, "vote_delegations_from_ids": []},
+        )
+        self.assert_model_exists(
+            "meeting_user/13",
+            {"vote_delegated_to_id": None, "vote_delegations_from_ids": [14]},
+        )
```

Comment:
Also add this test the other way around: With 14 delegating to 13 and the request payload being `{"vote_delegations_from_ids": [13], "vote_delegated_to_id": None}`
I am quite sure that that will have to be fixed as well